lerobot-web-be/app/utils/robots.py

```
@@ -0,0 +1,140 @@
+            if Path(target).name == dev_basename:
+                return Path(path).name  # "robot_leader_1"
+        except OSError:
+            continue
+    return None
+
+
+def configure_follower(
+    is_bi_setup: bool,
+    follower_map: dict,
+    robot_type: RobotType,
+    camera_config: dict = {},
```

Ruff (0.14.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)